expertsystem/data.py

```diff
@@ -3,11 +3,8 @@
 __all__ = [  # fix order in API
```

Comment:
This was added in order to (1) hide certain methods from the API and (2) fix the order of the definitions. It seems, however, that (2) fails somehow:
file:///mnt/c/dev/PWA/ComPWA/expertsystem/doc/_build/html/api/expertsystem.data.html#module-expertsystem.data

Not sure why it doesn't work anymore since 11d36d37e028bdd6faf922cec741de76db7a1c3f (upgrade from Sphinx 2.4.4 to 3.2.1), see also "order" in
https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directives